pkg/controller/storagecluster/reconcile.go

```diff
@@ -521,10 +522,10 @@ func (r *ReconcileStorageCluster) setRookUninstallandCleanupPolicy(instance *ocs
 	err = r.client.Get(context.TODO(), types.NamespacedName{Name: generateNameForCephCluster(instance), Namespace: instance.Namespace}, cephCluster)
 	if err != nil {
 		if errors.IsNotFound(err) {
-			reqLogger.Info("CephCluster not found, can't set the cleanup policy and uninstall mode")
+			reqLogger.Info("Uninstall: CephCluster not found, can't set the cleanup policy and uninstall mode")
 			return nil
 		}
-		return fmt.Errorf("Unable to retrive the cephCluster: %v", err)
+		return fmt.Errorf("Uninstall: Unable to retrive the cephCluster: %v", err)
```

Comment:
Unrelated to this patch but there is a typo. "retrive" should be "retrieve"

Reply:
done